packages/transition-backend/src/tasks/disseminationBlocks/importDisseminationBlocks.ts

```diff
@@ -0,0 +1,384 @@
+ *
+ * This file is licensed under the MIT License.
+ * License text available at https://opensource.org/licenses/MIT
+ */
+import inquirer from 'inquirer';
+import inquirerFileTreeSelection from 'inquirer-file-tree-selection-prompt';
+
+import { v4 as uuidV4 } from 'uuid';
+import censusQueries from '../../models/db/census.db.queries';
+import { GenericTask } from 'chaire-lib-common/lib/tasks/genericTask';
+import DataSource from 'chaire-lib-common/lib/services/dataSource/DataSource';
+import { ZoneAttributes } from 'chaire-lib-common/lib/services/zones/Zone';
+import { fileManager } from 'chaire-lib-backend/lib/utils/filesystem/fileManager';
+import dsQueries from 'chaire-lib-backend/lib/models/db/dataSources.db.queries';
+import zonesQueries from 'chaire-lib-backend/lib/models/db/zones.db.queries';
+import { parseCsvFile } from 'chaire-lib-backend/lib/services/files/CsvFile';
+import xmlReader from 'xml-reader';
+import fs from 'fs';
+
+// The structure of the object given by the xml-reader package. Copied from the npm page.
+interface XmlNode {
+    /** element name (empty for text nodes) */
+    name: string;
+    /** node type (element or text), see NodeType constants */
+    type: string;
+    /** value of a text node */
+    value: string;
+    /** reference to parent node (null with parentNodes option disabled or root node) */
+    parent: XmlNode;
+    /** map of attributes name => value */
+    attributes: { [name: string]: string };
+    /** array of children nodes */
+    children: XmlNode[];
+}
+
+interface ProximityIndices {
+    /** Normalised value of a dissemination block's proximity to employment. */
+    prox_idx_emp: number | null;
+    /** Normalised value of a dissemination block's proximity to pharmacy and drug stores. */
+    prox_idx_pharma: number | null;
+    /** Normalised value of a dissemination block's proximity to child care facilities. */
+    prox_idx_childcare: number | null;
+    /** Normalised value of a dissemination block's proximity to health facilities. */
+    prox_idx_health: number | null;
+    /** Normalised value of a dissemination block's proximity to grocery stores. */
+    prox_idx_grocery: number | null;
+    /** Normalised value of a dissemination block's proximity to primary education facilities. */
+    prox_idx_educpri: number | null;
+    /** Normalised value of a dissemination block's proximity to secondary education facilities. */
+    prox_idx_educsec: number | null;
+    /** Normalised value of a dissemination block's proximity to libraries. */
+    prox_idx_lib: number | null;
+    /** Normalised value of a dissemination block's proximity to parks. */
+    prox_idx_parks: number | null;
+    /** Normalised value of a dissemination block's proximity to transit trips. */
+    prox_idx_transit: number | null;
+}
+
+/**
+ * Task to import data about canada's dissemination blocks into the database.
+ * This is intended to be used with data provided by statcan.
```

Comment:
Can you provide links to the data and how to get the files or the specs of the file to import?